<ui version="4.0" >
 <class>sexselDlg</class>
 <widget class="QDialog" name="sexselDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Sex Selection</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QPushButton" name="maleBtn" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>Male</string>
    </property>
   </widget>
   <widget class="QPushButton" name="femaleBtn" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>Female</string>
    </property>
   </widget>
   <widget class="QPushButton" name="noneBtn" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>130</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>Unsexed</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Select the sex...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
